products/opentofu.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Opentofu
+category: app
+tags: cncf
+iconSlug: opentofu
+permalink: /opentofu
+versionCommand: tofu --version
+releasePolicyLink: https://support.hashicorp.com/hc/en-us/articles/360021185113-Support-Period-and-End-of-Life-EOL-Policy # currently same as hashicorp release policy
```

Comment:
Where this can be confirmed ?

Reply:
> release policy discussion, https://github.com/opentofu/opentofu/blob/main/TSC_SUMMARY.md#2024-01-22-async

that is the current decision

Reply:
i might take some time to evolve as community project

Reply:
It would be great to document this in the description, and remove the `releasePolicyLink` as it can be quite confusing IMO.

Reply:
can you propose the desired the change? I can adjust the PR accordingly.

Reply:
Something like this: https://github.com/endoflife-date/endoflife.date/pull/4756/commits/8b90853e0ddacec171aef2fced1b15d72cf1b1ac.

Reply:
I see, make sense.